crates/ar-cli/src/inspect.rs

```diff
@@ -0,0 +1,29 @@
+use bip39::{Language, Mnemonic};
+
+use crate::{common::{deserialize_pub_key, mnemonic_to_seed, seed_to_keypair, serialize_pub_key}, error::Error};
+
+
+
+
+pub fn run_mnemonic(maybe_mnemonic: &str) -> Result<(), Error> {
+    let mnemonic = Mnemonic::from_phrase(maybe_mnemonic, Language::English)?;
+    let seed = mnemonic_to_seed(&mnemonic);
+    let (_private_key, public_key) = seed_to_keypair(&seed);
```

Comment:
maybe we could print private key as well

Reply:
We could, but the intention here is to rely just on the mnemonic, we don't want to ask the AR to save the private key.